### DISCUSSION TITLE
【羊村】AILake 公告栏

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#236] [Announcements]
# AILake 公告栏

## 【AILake】PC/Web/手机端 AILake 上线了

- ~手机/PC端下载地址: <https://beta.ailake.cn>~
- 免费WEB地址:  <https://al.ibooker.org.cn>
- 免费｜AI画图
- 免费｜AI助手
- 免登录，免注册

## 【AILake 探索群】

- 交流探索 AI 技术，包括但不限于 LLM，GPT，AI绘画，TTS，STT，应用落地等等。
- 交流 VR/AR 技术
- 加群，私聊片刻微信: 529815144

## 【AILake 问题收集】

- 直接留言(本页面)，或者加群 at 片刻
- 留言板：<https://github.com/orgs/apachecn/discussions/236>

> 注意1：如果排版问题

- 可以使用chrome，但别屏蔽网站广告

> 注意2：如果灵力用完

- 可以使用chrome无痕模式(无限续杯)

**Comment by jiangzhonglian** [comment]
# 公告更新时间： 2024-04-11

当前不能提供 白嫖 了

想起那天夕阳下的奔跑，那是我逝去的青春 ...

一不小心我们已经免费近 11 个月辣 ... ^_^ ...

山高水远，江湖再见 ...

有条件的朋友，请多多支持 [OpenAI 官方](https://chat.openai.com) : <https://chat.openai.com> 哟 ~ ...^_^

[点击前往查看 -> 全网热点](https://www.ibooker.org.cn/hot): <https://www.ibooker.org.cn/hot>


---

## 2024-04-01：OpenAI 策略更新说明

OpenAI 在 2024/04/01 将之前的账号过期，试用金余额清空

## 2024-03-21：OpenAI 策略更新说明

OpenAI 在 2024/03/21 更新了新账号试用金策略，新账号无法再白嫖

## 2023-12-05：各平台热门新闻

- <https://al.ibooker.org.cn/hot>

![image](https://github.com/apachecn/.github/assets/9199175/7450f96b-2bd8-4937-88b0-ebfe329811e3)

## 2023-11-17：OpenAI 策略更新说明

OpenAI 在 2023/11/17 更新了请求策略，大规模封号和封IP

## 2023-08-22：Stable Diffusion 在线涂鸦绘画

快快快！涂鸦他来了！
在线涂鸦已上线测试啦！
有问题或建议麻烦随时艾特我

- <https://al.ibooker.org.cn/paint/app/scribble>

![image](https://github.com/apachecn/.github/assets/9199175/d21955ea-2bf6-4010-aaf8-66915cec0f82)


## 2023-08-18：Stable Diffusion 绘画提示词

 Stable Diffusion 提示词已经上线，
妈妈再也不用担心我不会构图了，需要什么点一点就好了 。

- <https://al.ibooker.org.cn/paint/prompt>

![image](https://github.com/apachecn/.github/assets/9199175/3e5b20d1-9f3b-47f7-b6b4-0598436310de)


## 2023-07-19：Stable Diffusion 绘画功能上线

刚上线基本功能很粗糙，勉强能用，消耗为 : 0.1灵石/1张图
持续迭代更新中 。。。


## 2023-07-15：OpenAI 策略更新说明

OpenAI 更新了请求策略，限制普通账号的API为200次/天，我们无法再像以前一样提供【无限制白嫖】了。
一不小心我们已经免费近 3 个月辣 ... ^_^ ...
还有需求使用的，当前已开启限制访问，每天可使用（MAX）20次。
兄弟萌且行且珍惜 。。。

**Comment by IndifferentArea** [comment]
您好，请问在我的浏览器上渲染成这种小窗口，这正常吗
![image](https://github.com/apachecn/.github/assets/74698401/1fd98ad3-6246-4adb-9d64-69b9259e39f1)

**Reply by jiangzhonglian** [reply]
好兄弟，你这个是不正常的，请用 chrome 无痕模式（别用那个广告拦截器 - 就会显示正常）

**Reply by jiangzhonglian** [reply]
别拦截广告，就正常显示

<img width="1220" alt="image" src="https://github.com/apachecn/.github/assets/9199175/98b75611-4600-4368-9fbf-a0d70184366f">

**Reply by IndifferentArea** [reply]
okok我懂了，谢谢

**Comment by jiangzhonglian** [comment]
1
